pkg/private/zip/build_zip.py

```diff
@@ -27,6 +27,7 @@
 # Unix dir bit and Windows dir bit. Magic from zip spec
 UNIX_DIR_BIT = 0o40000
 MSDOS_DIR_BIT = 0x10
+UNIX_SYMLINK_BIT = 0o120000
```

Comment:
I took it from the zip spec rather than stat.S_IFLNK because the API is zip.
Yes, it is identical, but the zip spec does not explicitly say it matches the posix bits.